source/tutorials/test-strategy/test-strategy.rst

```diff
@@ -0,0 +1,77 @@
+.. _system-validation-test-strategy:
+
+System Validation - Test strategy for |C|
+#########################################
+
+|C| is an open source Android software reference stack for the Android community to bring ideas to life while developing on the latest Android release and the latest Intel hardware platforms. The following |NUC| systems are used for validating regular |C| releases.
```

Comment:
Line 6 (second sentence) suggested rewrite: 
... The following |NUC| systems are used to validate regular |C| releases:

Reply:
fixed & thanks. :)